src/getPastes.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+
```

Comment:
Could separate types (Pastes) into separate module then import it when necessary